inst/rmarkdown/templates/trundler_template/skeleton/skeleton.Rmd

```diff
@@ -3,74 +3,117 @@ title: "Trundler Report Template"
 output: html_document
 ---
 
+<style type="text/css">
+  body{
```

Comment:
- Don't indent the `body` selector.
- Put a space between the selector and the opening `{`.